### DISCUSSION TITLE
MS2 spectra not read out fromThermo data

### DISCUSSION THREAD

**Discussion by JM-Bader** [vdemichev/DiaNN#488] [Q&A] [answered]
I want to process DIA data acquired on Thermo Q Exactive HF-X and Exploris480 (no FAIMS). 
However, DIANN 1.8.1 does not recognize the MS2 information and tells me that MS2 data is missing in the raw files. 
Likewise, I cannot convert the raw files to .dia. 
I have put 20 raw files in (all produce the error) which I know are not corrupted because I have processed them with Spectronaut before. 

How to fix the problem?

![image](https://user-images.githubusercontent.com/98888664/187896603-53fde4ee-3a42-47a5-b224-330984e5ba77.png)

**Comment by vdemichev** [comment, marked as answer]
Need to install MSFileReader on that PC using the link in the docs

**Reply by JM-Bader** [reply]
it works. Take care to exactly get the specific version listed in the DIA-NN docs. 
to download it from thermo, one may register a demo account (see demo tab) to access the software permanently. No need to specify the purchase/sale ID for a Thermo mass spectrometer that one has bought (for 'regular' registration to get the software)

**Comment by YuL-BI** [comment]
Hi there,
I got the same error message:  No MS2 spectra.  I have the right version of MSFileReader installed, and I also tried to copy the file to the DIA-NN folder, but still got the error message.  Could you please help?
Thanks very much.

**Reply by vdemichev** [reply]
Are the runs MSX-DIA?

**Reply by YuL-BI** [reply]
No, it was a simple DIA data.

**Reply by vdemichev** [reply]
The only two known causes of Thermo data not working so far: (i) either some fancy unsupported data like MSX or (ii) MSFileReader is isntalled not by the link provided in DIA-NN docs. Checking on some public DIA data canhelp determine whcih one is the case I guess...

**Comment by YuL-BI** [comment]
Dear Vadim,

The MSFilereader is as below which I downloaded again from the link (MS File Reader 3.0 SP3):

MSFileReader_x86_x64

I then copied it to the DIA-NN folder.  Is it correct?

I have tried another raw file which worked on another PC, but failed on my PC. Puzzled.  IS there anything else to check?

Thanks very much.

Best wishes,

Lu

**Reply by vdemichev** [reply]
It's an installer by the link, need torun it and it installs the MSFileReader in the system

**Comment by YuL-BI** [comment]
Hi Vadim

Stupid me 😊.  I will try to install it – need to sort out the admin right first.

Thank you very much!

Lu
